libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -241,6 +241,62 @@ void AP_Camera_Backend::send_camera_settings(mavlink_channel_t chan) const
         NaN);               // focusLevel float, percentage from 0 to 100, NaN if unknown
 }
 
+#if AP_CAMERA_SEND_FOV_STATUS_ENABLED
+// send camera field of view status
+void AP_Camera_Backend::send_camera_fov_status(mavlink_channel_t chan) const
+{
+    // getting corresponding mount instance for camera
+    AP_Mount* mount = AP::mount();
```

Comment:
```suggestion
    const AP_Mount* mount = AP::mount();
```

needs a nullptr check here

Reply:
Need to `const` the poi-getting-method in the mount library so it can be added here

Reply:
done